akka-remote/src/main/scala/akka/remote/artery/compress/ActualCompressionTables.scala

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.remote.artery.compress
+
+import akka.actor.ActorSystem
+import akka.event.{ Logging, LoggingAdapter }
+import akka.remote.artery.LiteralCompression
+
+/** INTERNAL API */
+object ActualCompressionTables {
```

Comment:
This class will go away in the form it exists right now I think, likely better to split up into incoming/outgoing (note that internally it is anyway already).